Automattic/jetpack
# Search: Use API values for paid allowance limits shown in the new pricing table

#### Changes proposed in this Pull Request:

Make certain values API-driven within the new pricing table.

<img width="1132" alt="image" src="https://user-images.githubusercontent.com/4044428/196266911-02313a9d-414a-46e1-85e6-590c68bd675e.png">

<img width="1149" alt="image" src="https://user-images.githubusercontent.com/4044428/196278175-9f24ecc7-d2be-4fdf-8781-fe15e5aad95c.png">


#### Other information:


#### Jetpack product discussion


p1666032314887379-slack-C02ME06LF?thread_ts=1666026581.553339&cid=C02ME06LF

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






Changing the API:
* Apply this change to your Jetpack site without a Search subscription.
* Sandbox your site's API connection using the `JETPACK__SANDBOX_DOMAIN` constant.
* Try changing the values returned for `monthly_search_request_limit`, `record_limit`, `quantity_per_unit`, and `per_unit_fee` in the pricing API.
* Ensure the new values are shown in the pricing table (`/wp-admin/admin.php?page=jetpack-search&new_pricing_202208=1`).

Changing the selector:
 * Apply this change to your Jetpack site without a Search subscription.
 * Change the selector code (e.g. `getPaidRequestsLimit: state => 40000`) to force a different value.
 * Ensure that different values are shown in the pricing table (`/wp-admin/admin.php?page=jetpack-search&new_pricing_202208=1`).
